relational/table.py

```diff
@@ -1067,6 +1069,24 @@ def _is_100_affordable(self, rows, proj):
                         name='is_100pct_affordable',
                         value='TRUE' if threshold_affordable else 'FALSE',
                         data=OEWDPermits.OUTPUT_NAME))
+                else:
+                    units = bmr = None
+                    try:
+                        units = int(proj.field('number_of_units',
+                                               Planning.NAME))
+                        bmr = int(proj.field('number_of_affordable_units',
+                                             Planning.NAME))
+                    except ValueError:
+                        return
+
+                    if units and bmr:
+                        threshold_affordable = \
```

Comment:
Can you add a comment explaining why we're doing this

Reply:
Done